modules/installing-ocp-agent.adoc

```diff
@@ -127,6 +129,10 @@ sshKey: |
 You must provide the IP address when you do not specify the node's IP addresses in the `networkConfig` parameter. If this address is not provided, one IP address is selected from the provided nodes's `networkConfig`.
 <2> The number of hosts defined must match the total number of hosts defined in the `install-config.yaml` file, which is the sum of the values of the `compute.replicas` and `controlPlane.replicas` parameters. When 3 master nodes and 0 worker nodes are defined in the `install-config.yaml` file,
 the number of hosts defined is 3. When 3 master nodes and 2 worker nodes are defined in the `install-config.yaml` file, the number of hosts defined is 5.
+<3> Optional: The `hostname` parameter is obtained either from the Dynamic Host Configuration Protocol (DHCP) or a reverse DNS lookup.
+<4> The `rootDeviceHints` parameter enables provisioning of the Red Hat Enterprise Linux CoreOS (RHCOS) image to a particular device. It examines the devices in the order it discovers them, and compares the discovered values with the hint values. It uses the first discovered device that matches the hint value.
+<5> Set this optional parameter to configure the network interface of a host. With NMState, you can configure various network interface types, DNS, and routing on cluster nodes.
```

Comment:
Maybe "Set this optional parameter to configure the network settings for the host, in NMState format."?
It might pay to link to https://nmstate.io/examples.html in additional resources as well.

Reply:
Done